files/common.sh

```diff
@@ -7,16 +7,16 @@ fail() {
   # Print a stderr: entry if there were anything printed to stderr
   if [[ -s $_tmp ]]; then
     # Hack to try and output valid json by replacing newlines with spaces.
-    echo "{ \"status\": \"error\", \"message\": \"$1\", \"stderr\": \"$(tr '\n' ' ' <$_tmp)\" }"
+    $(/usr/bin/which echo) "{ \"status\": \"error\", \"message\": \"$1\", \"stderr\": \"$(tr '\n' ' ' <$_tmp)\" }"
```

Comment:
What is the value of doing the `which echo` here? The addition to the PATH should fix the issue with the nonabsolute path. We can probably remove all of these `which` instances and just add the following to the top of `common.sh`.

```
export PATH=/usr/local/bin/:/bin/:/usr/bin:$PATH
```

Reply:
`echo` is a bash builtin, so there should be no need for `which` or `type`.